pkg/envoy/proxy_test.go

```diff
@@ -73,4 +73,28 @@ func TestEnvoyArgs(t *testing.T) {
 	}
 }
 
-// TestEnvoyRun is no longer used - we are now using v2 bootstrap API.
+func TestSplitComponentLog(t *testing.T) {
+	cases := []struct {
+		input      string
+		log        string
+		components []string
+	}{
+		{"info", "info", nil},
+		// A bit odd, but istio logging behaves this way so might as well be consistent
+		{"info,warn", "warn", nil},
+		{"info,misc:warn", "info", []string{"misc:warn"}},
+		{"misc:warn,info", "info", []string{"misc:warn"}},
```

Comment:
This kind of flags are very hard for users to set, could you add some valid examples to flag help doc

```
	proxyCmd.PersistentFlags().StringVar(&proxyLogLevel, "proxyLogLevel", "warning",
		fmt.Sprintf("The log level used to start the Envoy proxy (choose from {%s, %s, %s, %s, %s, %s, %s})",
			"trace", "debug", "info", "warning", "error", "critical", "off"))
	proxyCmd.PersistentFlags().IntVar(&concurrency, "concurrency", 0, "number of worker threads to run")
	// See https://www.envoyproxy.io/docs/envoy/latest/operations/cli#cmdoption-component-log-level
	proxyCmd.PersistentFlags().StringVar(&proxyComponentLogLevel, "proxyComponentLogLevel", "misc:error",
		"The component log level used to start the Envoy proxy")
```

Reply:
thanks, updated the docs